modules/security-context-constraints-psa-opting.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-psa-opting_{context}"]
+= Controlling pod security admission synchronization
+
+You can enable or disable automatic pod security admission synchronization for most namespaces.
+
+[NOTE]
+====
+By default, namespaces that have the prefix 'openshift-' have pod security admission synchronization disabled. 
+
+Namespaces that the installer creates have pod security admission synchronization disabled permanently.
```

Comment:
@stlaz it is not true. See that `openshift-operators` is created by the installer and allows the PSA because it is not a OCP playload namespace:

```suggestion
Namespaces that are defined as OCP payloads have pod security admission synchronization disabled permanently. You can check the list of those namespaces [here](https://github.com/openshift/cluster-policy-controller/blob/release-4.11/pkg/psalabelsyncer/podsecurity_label_sync_controller.go#L42-L111).
```

Reply:
I believe the docs should never have links to the code.

Also, yes, the `openshift-operators` is unfortunately special, I forgot about it, again :|

Reply:
```suggestion
Namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. For a reference for these namespaces, see "List of payload namespaces".
```

Added document `modules/security-context-constraints-psa-payload-namespaces.adoc`. We should not be linking to code whenever possible.

Reply:
Hi @maxwelldb: 
We can add a doc with the list of the namespace however, we cannot say

` Namespaces that the installer creates ` Not all namespaces prefixed with "openshift" created by the installer are part of the OCP Playload and have the limitation that will never have the label syncer working there.

Reply:
> Namespaces that the installer creates have pod security admission synchronization disabled permanently.

This was the better version, nobody says that every namespace prefixed with "openshift-" was created by the installer.

Possibly add `(with the exception of openshift-operators namespace)` somewhere

Reply:
```suggestion
Namespaces that the installer creates have pod security admission disabled permanently. These namespaces include:

* All namespaces that are prefixed with `openshift-`, except for `openshift-operators`
* `default``
* `kube-node-lease`
* `kube-system`      
* `kube-public`      
* `openshift`
```